### ISSUE TITLE
windows/nodejs error

### ISSUE THREAD

**Issue by johntom** [delaneyj/realworld-datastar#4]
win10 node 22.1.0 go 123.3
when I run
go run main.go
the following error is raised
..\..\sql\setup.go:17:2: no required module provides package github.com/delaneyj/realworld-datastar/sql/zz; to add it:
        go get github.com/delaneyj/realworld-datastar/sql/zz
when I run the go get...
the following message
go: github.com/delaneyj/realworld-datastar/sql/zz: no matching versions for query "upgrade"

**Comment by johntom** [comment]
Please provide setup instructions in readme to complete code to run the demo as I'm new to go and Obviously missing some steps. Also, I think a guide to environment vairable paths will be a great help. On my end  I just enrolled in some Udemy go courses to assist in learning this technology stack.

**Comment by bogdano** [comment]
I don't think that Node has anything to do with this? There seems to be an issue with the sqlc-gen-zombiezen tool; this is what I get when I run task -w:

```
task: Started watching for tasks: default
task: [kill] killall -q realworld_bin  || echo "Process was not running."
task: [sqlc] sqlc generate
task: Task "templ" is up to date
Process was not running.
# package zz
error generating code: process: sqlc-gen-zombiezen not found
exit status 1
```
